lib/python/treadmill/sproc/tickets.py

```diff
@@ -151,7 +151,9 @@ def fetch(appname, tkt_spool_dir, lockers, sleep):
                 tickets.request_tickets_from(
                     host, port, appname, tkt_spool_dir
                 )
-
+            # Invoking with --sleep 0 will fetch tickets once and exit.
```

Comment:
[nit] Should be an admin command then.. I don't think sproc as CLI is a right way to go.  But this is bigger refactor